.latexmkrc

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env perl
-$latex            = "find . -type f -name '*.tex' -print0 | xargs -0 sed -i -e 's/、/，/g' -e 's/。/．/g' && uplatex -synctex=1 -interaction=nonstopmode -halt-on-error";
-$latex_silent     = "find . -type f -name '*.tex' -print0 | xargs -0 sed -i -e 's/、/，/g' -e 's/。/．/g' && uplatex -synctex=1 -interaction=batchmode -halt-on-error";
-$bibtex           = "upbibtex";
-$biber            = "biber --bblencoding=utf8 -u -U --output_safechars";
+$latex            = "find . -type f -name '*.tex' -print0 | xargs -0 sed -i -e 's/、/，/g' -e 's/。/．/g' && uplatex %O -synctex=1 -halt-on-error -interaction=nonstopmode -shell-escape -file-line-error %S";
```

Comment:
(this is not a request, but a comment)

I didn't know `-file-line-error` option, but it's a nice option.
However, the current code for coloring don't support the error output format, so I opened a new issue: https://github.com/arkark/latexmk-docker/issues/8.